synth.py

```diff
@@ -18,9 +18,9 @@
 from synthtool import gcp
 
 gapic = gcp.GAPICBazel()
+common = gcp.CommonTemplates()
 
-versions = ["v2beta1", "v2"]
-
+versions = ["v2"]
```

Comment:
Please re-add the v2beta1. (Some service teams continue to use the beta to test out new features before adding them to the stable version.)